static/js/components/tiles/top_event_tile.tsx

```diff
@@ -40,7 +40,7 @@ import { rankingPointsToCsv } from "../../utils/chart_csv_utils";
 import { getPlaceNames } from "../../utils/place_utils";
 import { ChartFooter } from "./chart_footer";
 
-const RANKING_COUNT = 10;
+const RANKING_COUNT = 20;
```

Comment:
did you check how it looks on NL interface?

Reply:
as discussed offline, made this an optional property of top event tile spec